Sources/Extensions/CALayer+Shadow.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CALayer+Shadow.swift
+//  iOSSnapshotTestCase
+//
+//  Created by Hannes Van den Berghe on 28/11/2018.
+//
+
+import UIKit
+
+extension CALayer {
+    
+    // swiftlint:disable identifier_name
+    public func applySketchShadow(color: UIColor = .black,
+                                  alpha: Float = 0.1,
```

Comment:
Alpha zit toch in de kleur bij Sketch, niet?

Reply:
Heb het hernoemd naar opacity die nodig is om de layer op te bouwen. De alpha kan je inderdaad in de kleur zetten.

Reply:
Okay top!